### DISCUSSION TITLE
Where do I find the subscription for telemetry validation?

### DISCUSSION THREAD

**Discussion by briancjohns** [google/digitalbuildings#1181] [Q&A]
The Instance Validator requires a fully qualified path to a Google Cloud PubSub subscription in order to do telemetry validation.  Where do I find the path for my particular project?

https://github.com/google/digitalbuildings/tree/master/tools/validators/instance_validator#telemetry-validation

**Comment by tasodorff** [comment]
please reach out to your IoT TPM for this information if needed, or to bos-gps@ for more support with your specific project.
